packages/plugin-vue/src/handleHotUpdate.ts

```diff
@@ -44,8 +44,8 @@ export async function handleHotUpdate(
       return m1.url.length - m2.url.length
     })[0]
   const templateModule = modules.find((m) => /type=template/.test(m.url))
-
-  if (hasScriptChanged(prevDescriptor, descriptor)) {
+  const scriptChanged = hasScriptChanged(prevDescriptor, descriptor)
+  if (scriptChanged) {
```

Comment:
```suggestion

  const scriptChanged = hasScriptChanged(prevDescriptor, descriptor)
  if (scriptChanged) {
```
Let's preserve this empty line.